apps/api/src/app/workflows-v2/util/template-parser/liquid-parser.ts

```diff
@@ -0,0 +1,126 @@
+import { Template, Liquid, RenderError, LiquidError } from 'liquidjs';
```

Comment:
Do you think this should live in application generic package?

Reply:
IMO once we reuse it in another place, we should move it to app-gen.